src/bokeh/models/plots.py

```diff
@@ -554,6 +555,17 @@ def _check_bad_extra_range_name(self) -> str | None:
     .. note:: This feature is experimental and may change in the short term.
     """)
 
+    window_axis = Nullable(Enum(WindowAxis), default=None, help="""
```

Comment:
This has to be added to the respective `*.pyi` file.

Reply:
7533c21